packages/ui/src/components/va-file-upload/VaFileUploadUndo/VaFileUploadUndo.vue

```diff
@@ -1,6 +1,14 @@
 <template>
-  <div class="va-file-upload-undo">
-    <span>File was successfully deleted</span>
+  <div
+    class="va-file-upload-undo"
+    :class="{ 'va-file-upload-undo--vertical': $props.vertical }"
+  >
+    <va-progress-bar
+      :model-value="progress"
+      :rounded="false"
+      class="va-file-upload-undo__progress-bar"
+    />
+    <span class="va-file-upload-undo__text">File was successfully deleted</span>
```

Comment:
```suggestion
    <span class="va-file-upload-undo__text">{{ props.fileDeletedText }}</span>
```
we have to add this prop (name as example)